src/cider/nrepl/middleware/version.clj

```diff
@@ -0,0 +1,28 @@
+(ns cider.nrepl.middleware.version
+  "Return version info of the CIDER-nREPL middleware itself."
+  (:require [clojure.tools.nrepl.middleware :refer [set-descriptor!]]
+            [cider.nrepl.version :as version]
+            [cider.nrepl.middleware.util.error-handling :refer [with-safe-transport]]))
+
+(defn- cider-version-reply
+  "Returns CIDER-nREPL's version as a map which contains `:major`,
+  `:minor`, `:incremental`, and `:qualifier` keys, just as
+  `*clojure-version*` does."
+  [msg]
+  {:cider-version version/version})
+
+(defn wrap-version
+  "Middleware that provides code formatting."
```

Comment:
Lololol!! Pitfalls of copy paste
On Apr 10, 2016 12:52 AM, "Bozhidar Batsov" <notifications@github.com>
wrote:

> In src/cider/nrepl/middleware/version.clj
> <https://github.com/clojure-emacs/cider-nrepl/pull/334#discussion_r59127053>
> :
>
> > @@ -0,0 +1,28 @@
> > +(ns cider.nrepl.middleware.version
> > +  "Return version info of the CIDER-nREPL middleware itself."
> > +  (:require [clojure.tools.nrepl.middleware :refer [set-descriptor!]]
> > +            [cider.nrepl.version :as version]
> > +            [cider.nrepl.middleware.util.error-handling :refer [with-safe-transport]]))
> > +
> > +(defn- cider-version-reply
> > +  "Returns CIDER-nREPL's version as a map which contains `:major`,
> > +  `:minor`, `:incremental`, and `:qualifier` keys, just as
> > +  `*clojure-version*` does."
> > +  [msg]
> > +  {:cider-version version/version})
> > +
> > +(defn wrap-version
> > +  "Middleware that provides code formatting."
>
> code formatting? :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/clojure-emacs/cider-nrepl/pull/334/files/137094028ede4d6450bc12c854b2acebb985f212#r59127053>
>